cdap-docs/reference-manual/source/glossary.rst

```diff
@@ -63,6 +63,11 @@ Glossary
       synchronous calls into CDAP from an external system and perform server-side
       processing on-demand. They are similar to a stored procedure in a traditional database
       system. Procedures are typically used to post-process data at query time.
+
+   Worker
+      Workers are typically long running programs that can be used to execute tasks. Each instance
```

Comment:
Workers are typically long-running background programs that can be used to execute tasks. Each instance of a worker runs either in its own YARN container (CDAP distributed mode) or a single thread (CDAP Standalone or In-Memory mode) and the number of instances may be updated via the
CDAP Console, RESTful APIs or the CLI. Datasets can be accessed from inside Workers.